en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -3,8 +3,12 @@
 Choreo Connect Analytics provides reports, dashboards, statistics, and graphs for the APIs deployed on Choreo Connect.
 WSO2 Choreo Connect has the capability to publish events to the Choreo platform in order to generate analytics. This page describes the feature and explains how the feature could be used to generate useful analytics in order to gain important insights into the APIs deployed on the Choreo Connect. To learn more concepts on analytics, follow the [concepts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics/).
 
+There are two ways you can configure Analytics,
 
-## Configuring Analytics for Choreo Connect
+1. [Choreo Portal Analytics for Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/#choreo-portal-analytics-for-choreo-connect)
+2. [ELK Analytics for Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/#elk-analytics-for-choreo-connect)
```

Comment:
Links within the same page can be shortened like this https://github.com/wso2/docs-apim/blob/63829dc504aff1ff82e7a360c0dae0ba53d2e23a/en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md?plain=1#L13